src/Symfony/Component/Form/Form.php

```diff
@@ -416,6 +416,13 @@ public function handleRequest(mixed $request = null): static
         return $this;
     }
 
+    public function isSubmittedRequestValid(mixed $request = null): bool
```

Comment:
I like this method; it simplifies three operations in one call and makes sense to me to relate them as part of the Form component. But, I would typehint `Request` directly here, I don't see any blockers to not doing it; do you?